archive/test/ArchiveControllerTest.scala

```diff
@@ -6,16 +6,18 @@ import play.api.test.Helpers._
 import org.scalatest.{BeforeAndAfterAll, DoNotDiscover, FlatSpec, Matchers}
 
 import scala.concurrent.Future
-import services.DynamoDB
+import services.Redirects
 
 @DoNotDiscover class ArchiveControllerTest
   extends FlatSpec
   with Matchers
   with ConfiguredTestSuite
-  with BeforeAndAfterAll
-  with WithTestWsClient {
+  with BeforeAndAfterAll {
 
-  lazy val archiveController = new ArchiveController(new DynamoDB(wsClient))
+  lazy val archiveController = new ArchiveController(mockRedirects)
+  lazy val mockRedirects = new Redirects {
+    override def destinationFor(source: String) = Future.successful(None)
+  }
```

Comment:
Planning to create a `WithTestDynamoClient` trait or similar which will mean this mocking isn't required, but I'll do that in another PR.  This one is already too bloated.